lib/request-proxy.js

```diff
@@ -26,6 +26,8 @@ var TypedError = require('error/typed');
 
 var safeParse = require('./util').safeParse;
 
+var RETRY_SCHEDULE = [0, 0.5, 2];
```

Comment:
it very well could, but maybe initially we be a little bit more conversative and bump up the max to 2+